addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -551,8 +545,12 @@ private void setProperties(Device device) {
         }
     }
 
-    /* (non-Javadoc)
-     * @see org.eclipse.smarthome.core.thing.binding.BaseThingHandler#bridgeStatusChanged(org.eclipse.smarthome.core.thing.ThingStatusInfo)
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
```

Comment:
I suppose this one should be removed as well.

@kaikreuzer we have 18 `(non-Javadoc)` docblock like things, for example this one and the following one:

```
    /*
     * (non-Javadoc)
     *
     * @see java.lang.Object#toString()
     */
```

Should this be removed and maybe added as a check as well

See also https://stackoverflow.com/questions/4597329/does-non-javadoc-have-a-well-understood-meaning